cmd/minikube/cmd/config/config.go

```diff
@@ -193,6 +194,10 @@ var settings = []Setting{
 		name: "disable-driver-mounts",
 		set:  SetBool,
 	},
+	{
+		name:   "cache",
```

Comment:
This also needs a set method, since thats what `minikube config set` uses.  Otherwise, it will panic!

```diff
diff --git a/cmd/minikube/cmd/config/config.go b/cmd/minikube/cmd/config/config.go
index 15bad2c56..582027d5a 100644
--- a/cmd/minikube/cmd/config/config.go
+++ b/cmd/minikube/cmd/config/config.go
@@ -196,6 +196,7 @@ var settings = []Setting{
        },
        {
                name:   "cache",
+               set:    SetConfigMap,
                setMap: SetMap,
        },
 }
diff --git a/cmd/minikube/cmd/config/util.go b/cmd/minikube/cmd/config/util.go
index 5a2db1d7a..212771ae3 100644
--- a/cmd/minikube/cmd/config/util.go
+++ b/cmd/minikube/cmd/config/util.go
@@ -20,6 +20,7 @@ import (
        "fmt"
        "os"
        "strconv"
+       "strings"

        "github.com/pkg/errors"
        "k8s.io/minikube/pkg/minikube/assets"
@@ -65,6 +66,16 @@ func SetMap(m config.MinikubeConfig, name string, val map[string]interface{}) er
        return nil
 }

+func SetConfigMap(m config.MinikubeConfig, name string, val string) error {
+       list := strings.Split(val, ",")
+       v := make(map[string]interface{})
+       for _, s := range list {
+               v[s] = nil
+       }
+       m[name] = v
+       return nil
+}
+
 func SetInt(m config.MinikubeConfig, name string, val string) error {
        i, err := strconv.Atoi(val)
        if err != nil {
```